app/models/speaker.py

```diff
@@ -33,7 +33,7 @@ class Speaker(SoftDeletionModel):
     country = db.Column(db.String)
     city = db.Column(db.String)
     gender = db.Column(db.String)
-    rank = db.Column(db.Integer, default=0, nullable=False)
+    rank = db.Column(db.Integer, server_default='0', nullable=False)
```

Comment:
Where is default. server_default is not a valid sqlalchemy kwarg

Reply:
Please check now